pkg/select/blockdevice/select.go

```diff
@@ -58,10 +58,12 @@ func (c *Config) getCandidateDevices(bdList *apis.BlockDeviceList) (*apis.BlockD
 			FilterBlockDeviceName,
 		)
 	} else {
+		if c.ClaimSpec.DeviceType != "sparse" {
```

Comment:
Good idea to use the constants defined in 'github.com/openebs/node-disk-manager/blockdevice'.
```suggestion
		if c.ClaimSpec.DeviceType != blockdevice.SparseBlockDeviceType {
```